working/manual_changes/2024/2024-07-09.sql

```diff
@@ -0,0 +1,121 @@
+--add new UCUM concepts
+DO $_$
```

Comment:
Why not put all the calls to the AddNewConcept() function within a single DO block?

Reply:
Let's call it common practice/ habits /style ; Separate DO blocks were used for quite a while - so it's a matter of fact. I assume that it's unlikely that changing to a different approach would significantly impact the outcome or its associated cost.